mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
@percona-ysorokin Now, together with the recommendations the test is "rock solid". I have run it in the configuration you wrote couple of times, all seems to work fine. The test itself is based on other SBM tests and they work just fine. Now it is far more secured than the other tests.